exercises/B-boolean-literals/exercise.js

```diff
@@ -6,7 +6,9 @@
 */
 
 let codeYourFutureIsGreat = true;
-
+let mozafarIsCool = false;
+let calculationCorrect = true;
+let moreThan10Students = false;
```

Comment:
Including numbers as variable names may become an issue later on. Try avoiding using them please. 
Everything else is fine!